models/role.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Role = sequelize.define(
```

Comment:
realized that this was inconsistent too - we should stick with lowercase
```suggestion
  const role = sequelize.define(
```